base_images/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM amazonlinux:2
+
+# This image can be used to run unit tests for ORCA.
+# TODO: remove packages from tasks/lambda/requirements-dev.txt and tasks/lambda/requirements.txt 
+# except coverage, pytest, fastjsonschema and shared libraries- throws no module found error
+
+ENV NODE_VERSION="16.x"
+ENV TERRAFORM_VERSION "0.13.6"
+
+# Add NodeJS and Yarn repos & update package index
+RUN \
+curl -sL https://rpm.nodesource.com/setup_${NODE_VERSION} | bash - && \
+curl -sL https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo && \
+yum update -y
+
+# CLI utilities
+RUN yum install -y gcc git make openssl wget zip unzip tar
+
+# install Python 3 and pip
+RUN \
+yum install -y python3-devel && \
+yum install -y nodejs yarn && \
+yum -y install python3-pip && \
+pip3 install -q --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org
+
+# AWS & Terraform
+RUN \
+yum install -y awscli && \
+wget "https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip" && \
+unzip *.zip && \
+chmod +x terraform && \
+mv terraform /usr/local/bin
+
+# Install and configure git-secrets
```

Comment:
We should be able to remove git-secrets from this build since it will be used out in Bamboo land. Git Secretes is good to have when you are doing things locally.